invenio_rdm_records/services/communities/service.py

```diff
@@ -80,14 +192,14 @@ def remove(self, identity, record_id, data, revision_id=None, uow=None):
                 errors.append(
                     {
                         "community": community_id,
-                        "message": "The record does not belong to the community.",
+                        "message": _("The record does not belong to the community."),
```

Comment:
doubt: do these display correctly in the UI? in many places (e.g. validation) the error message(s) is a list (i.e. `messages`.

Reply:
Thanks! I will test this: the response format will change depending also on the UI. I will test integration and make changes.